mobly/suite_runner.py

```diff
@@ -77,12 +77,60 @@ def setup_suite(self, config):
 from mobly import records
 from mobly import signals
 from mobly import test_runner
+from mobly import utils
 
 
 class Error(Exception):
   pass
 
 
+class SuiteInfoRecord:
+  """A record representing the test suite info in test summary.
+
+  Attributes:
+    test_suite_class: str, the class name of the test suite class.
+    test_suite_version: str, the test suite version.
+    begin_time: int, epoch timestamp of when the suite started.
+    end_time: int, epoch timestamp of when the suite ended.
+  """
+
+  KEY_TEST_SUITE_CLASS = 'Test Suite Class'
+  KEY_TEST_SUITE_VERSION = 'Test Suite Version'
```

Comment:
Instead of hard coding fields one by one, should we provide a mechanism for users to dump their own info as needed.
Like in the test case record, the essential fields are hard coded, then there's the user defined info field

Reply:
Good point. Added a property `BaseSuite.suite_info` for user defined suite info.